app/views/devise/registrations/edit_password.html.erb

```diff
@@ -18,37 +18,24 @@
     <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do %>
     <%= render "devise/shared/error_messages", resource: resource %>
 
```

Comment:
Consider: adding a email/username field here to update password managers.
I couldn't figure out where the user's email would likely to be, so that would need setting.
```suggestion
      <%= email_field_tag :"user[email]", USERS_EMAIL, class: "govuk-!-display-none", autocomplete: "off", "aria-hidden": true %>
```